HelpfulItem.h

```diff
@@ -1,15 +1,12 @@
 #pragma once
 
-#include "Item.h"
 #include "Dwarf.h"
-#include "Character.h"
 #include "Paladin.h"
 #include "DragonSlayer.h"
 #include "Dragon.h"
```

Comment:
Don't include what you don't use!
None of the information defined in these header files are used in this header file.
- [ ] revise